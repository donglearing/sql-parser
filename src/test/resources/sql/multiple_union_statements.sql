select a from T union select b from F union select c FROM H where c in (select c from H1 union select c from H2);