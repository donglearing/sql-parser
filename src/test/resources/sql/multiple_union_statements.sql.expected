SELECT a FROM T UNION SELECT b FROM F UNION SELECT c FROM H WHERE c IN ((SELECT c FROM H1) UNION (SELECT c FROM H2))